CMakeLists.txt

```diff
@@ -395,44 +393,6 @@ if(MSVC)
   target_compile_options(gtest_main PRIVATE /Zc:__cplusplus)
 endif()
 
-# Concise way to register a new google test
```

Comment:
Why do we no longer need this ? Also, how will it effect the gtests upstream ?

Reply:
Because we reworked gtests and this is useless and do nothing.

Reply:
This was added back, will be looked in future gtest PR.